src/styles/modal.css

```diff
@@ -0,0 +1,20 @@
+.modal {
+    display: flex;
+    width: 100%;
+    height: 100%;
+    justify-content: center;
+    align-items: center;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(43, 43, 43, 0.9);
```

Comment:
Remove first background-color as it is getting overridden

Reply:
I added it as a fallback to second one